test/utils/image/manifest.go

```diff
@@ -84,7 +84,7 @@ var (
 	NoSnatTestProxy          = ImageConfig{e2eRegistry, "no-snat-test-proxy", "1.0", true}
 	NWayHTTP                 = ImageConfig{e2eRegistry, "n-way-http", "1.0", true}
 	// When these values are updated, also update cmd/kubelet/app/options/container_runtime.go
```

Comment:
per this comment, I think you need to update cmd/kubelet/app/options/container_runtime.go?

https://github.com/kubernetes/kubernetes/blob/6590ea6d5d50700d34255b1e037b2702ad26b7fc/cmd/kubelet/app/options/container_runtime.go#L36-L40